variable/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import "fmt"
+
+func main() {
+	//Normal Variable Declaration
```

Comment:
There is another way as well

```go
// using type and int
var i int = 10
// var inferred
var i = 10
// using colon inferred
i := 10
```

[https://github.com/aasisodiya/go/tree/master/golang-general/golang-basics#variable-declaration-in-go](https://github.com/aasisodiya/go/tree/master/golang-general/golang-basics#variable-declaration-in-go)

Reply:
@atharvaskadam did you read above comment?